addons/website/static/src/js/content/menu.js

```diff
@@ -135,6 +142,19 @@ const BaseAnimatedHeader = animations.Animation.extend({
         this.fixedHeader = useFixed;
         this._adaptToHeaderChange();
         this.el.classList.toggle('o_header_affixed', useFixed);
+        this._adaptFixedHeaderPosition();
+    },
+    /**
+     * @private
+     */
+    _updateMainPaddingTop: function () {
+        this.headerHeight = this.$el.outerHeight();
+        this.topGap = this._computeTopGap();
+
+        if (this.isOverlayHeader) {
+            return;
```

Comment:
Not sure what to do here. Before this PR, it fixed an issue on mobile when you open the burger menu, the content were pushed down depending on dropdown height. I know that this behavior was already present in 15.3 but I find this bug more restrictive than the one where the content overlaps on the scrollbar.
Maybe we can fix this with this return.

Reply:
We will fix this later. In any case, we need to set the position to `fixed` and therefore the compensate. It's why I will merge this PR anyway.